src/config.cpp

```diff
@@ -86,12 +128,39 @@ static inline void print_version(std::ostream& os)
 static inline void print_usage(std::ostream& os, const char* name,
                                const po::options_description& desc)
 {
+    static constexpr char argument_detail[] =
+        R"(
+Arguments to options:
+    PATH        A filesystem path.  It can be either relative to the
+                current working directory or absolute.
+
+    EVENT       Name of a perf event.  Format is either
+                - for a predefined event:
+                    <name>
+                - for a kernel PMU event one of:
+                    <pmu>/<event>/
+                    <pmu>:<event>
+                  Kernel PMU events can be found at
+                    /sys/bus/event_source/devices/<pmu>/event/<event>.
+
+    TRACEPOINT  Name of a kernel tracepoint event.  Format is either
+                    <group>:<name>
+                or
+                    <group>/<name>.
+                Tracepoint events can be found at
+                    /sys/kernel/debug/tracing/events/<group>/<name>.
+                Accessing tracepoints (even for --list-tracepoints)
+                usually requires read/execute permissions on
+                    /sys/kernel/debug
+)";
```

Comment:
This should also contain an entry for `KNOB` and `CLOCKID`.
Also, mention the list command for each one.